src/packages/web/page.gleam

```diff
@@ -87,11 +91,27 @@ fn package_list_item(package: PackageSummary) -> Node(t) {
   html.li(
     [],
     [
-      html.div_text(
-        [attrs.class("package-date-time")],
-        format_date(package.updated_in_hex_at),
+      html.div(
+        [],
+        [
+          html.h2(
+            [attrs.class("package-name")],
+            [external_link_text(url, package.name)],
+          ),
+          case latest_version {
+            Ok(latest_version) ->
+              html.h3_text(
```

Comment:
I don't think this is a heading semantically?

Reply:
Changed it to be a plain `div`.